include/flatbuffers/flexbuffers.h

```diff
@@ -337,6 +337,17 @@ class Map : public Vector {
   bool IsTheEmptyMap() const { return data_ == EmptyMap().data_; }
 };
 
+template<typename T>
+void AppendToString(std::string &s, T &&v, bool keys_quoted)
+{
+    s += "[ ";
+    for (size_t i = 0; i < v.size(); i++) {
+      v[i].ToString(true, keys_quoted, s);
+      if (i < v.size() - 1) s += ", ";
```

Comment:
```
flexbuffers.h:346:13: error: comparison between signed and unsigned integer expressions [-Werror=sign-compare]
       if (i < v.size() - 1) s += ", ";
```

Reply:
You can simply do `if (i)` if you move this statement to above `ToString` :)

Reply:
Thanks.. updated the new changes.
Just forgot about this PR, causing such a delay!

Reply:
@aardappel fixed the formatting case. I actually didn't correctly understood your comment that's why had this alternate fix.

Reply:
If you don't understand, ask, don't ignore :)

How about:
```
      if (i) s += ", ";
      v[i].ToString(true, keys_quoted, s);
```

Reply:
oh I see now. We can do either way. Will submit a patch with suggested changes. Thanks :)